.github/workflows/python-app.yml

```diff
@@ -66,10 +66,9 @@ jobs:
         ./install.sh
         conda install --prefix ./envs/Macrel_env  -y pytest
 
-    - name: Activating Macrel environment
-      run: source /usr/share/miniconda3/bin/activate /home/runner/work/macrel/macrel/envs/Macrel_env
-                      
     - name: Evaluate Macrel Build
       run: |
+        source /usr/share/miniconda3/bin/activate /home/runner/work/macrel/macrel/envs/Macrel_env
+        macrel="/home/runner/work/macrel/macrel/envs/Macrel_env/bin/macrel"
```

Comment:
is this `macrel=...` necessary? it does not seem to be used elsewhere

Reply:
it is needed because the command "which" could not find it by itself for some reason when Macrel_env was activated